ribbon-loadbalancer/src/main/java/com/netflix/client/ClientFactory.java

```diff
@@ -230,17 +233,18 @@ public static IClientConfig getNamedConfig(String name) {
      * is created and {@link IClientConfig#loadProperties(String)} will be called.
      */
     public static IClientConfig getNamedConfig(String name, Class<? extends IClientConfig> clientConfigClass) {
```

Comment:
public method no longer returns 'null' in the arbitrary IClientConfig failure case, but throws an exception instead.   Is the upcoming release not backwards-compatible?